.github/workflows/fetch-sbom.yml

```diff
@@ -0,0 +1,76 @@
+name: Weekly SBOM Update
+
+on:
+  schedule:
+    - cron: '30 18 * * 6' # Runs every Sunday at midnight IST (18:30 UTC on Saturdays)
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pages: write
+
+jobs:
+  fetch-sbom:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Get latest release versions for care and care_fe
+        id: versions
+        run: |
+          CARE_VERSION=$(curl -s https://api.github.com/repos/ohcnetwork/care/releases/latest | jq -r .tag_name)
+          CARE_FE_VERSION=$(curl -s https://api.github.com/repos/ohcnetwork/care_fe/releases/latest | jq -r .tag_name)
+          echo "CARE_VERSION=$CARE_VERSION" >> $GITHUB_ENV
+          echo "CARE_FE_VERSION=$CARE_FE_VERSION" >> $GITHUB_ENV
+
+      - name: Create SBOM directories for care and care_fe (if not exists)
+        run: |
+          if [ ! -d "care/${{ env.CARE_VERSION }}" ]; then
+            mkdir -p care/${{ env.CARE_VERSION }}
+          fi
+          if [ ! -d "care_fe/${{ env.CARE_FE_VERSION }}" ]; then
+            mkdir -p care_fe/${{ env.CARE_FE_VERSION }}
+          fi
```

Comment:
why check if directory exists when `-p` flag is used?

https://stackoverflow.com/a/793867